AdlsSDKGettingStarted/Program.cs

```diff
@@ -12,41 +13,53 @@ public class Program
+            // Create Client Secret Credential
+            var creds = new ClientSecretCredential(tenantId, applicationId, clientSecret);
 
-            // Create ADLS client object
-            AdlsClient client = AdlsClient.CreateClient(adlsAccountFQDN, clientCreds);
+            // Create data lake file service client object
+            DataLakeServiceClient serviceClient = new DataLakeServiceClient(new Uri(serviceUri), creds);
+            var name = "sample-filesystem" + Guid.NewGuid().ToString("n").Substring(0, 8);
+            // Create data lake file system client object
+            DataLakeFileSystemClient filesystemclient = serviceClient.GetFileSystemClient(name);
+            
+            filesystemclient.CreateIfNotExists();
 
             try
             {
+                long length;
                 string fileName = "/Test/testFilename1.txt";
+                DataLakeFileClient file = filesystemclient.GetFileClient(fileName);
 
-                // Create a file - automatically creates any parent directories that don't exist
-                // The AdlsOuputStream preserves record boundaries - it does not break records while writing to the store
-                using (var stream = client.CreateFile(fileName, IfExists.Overwrite))
+                // Upload a file - automatically creates any parent directories that don't exist
+                using (var stream = new MemoryStream())
                 {
                     byte[] textByteArray = Encoding.UTF8.GetBytes("This is test data to write.\r\n");
                     stream.Write(textByteArray, 0, textByteArray.Length);
 
                     textByteArray = Encoding.UTF8.GetBytes("This is the second line.\r\n");
                     stream.Write(textByteArray, 0, textByteArray.Length);
+                    length = stream.Length;
+                    stream.Seek(0, SeekOrigin.Begin);
+                    file.Upload(stream, true);
                 }
 
                 // Append to existing file
-                using (var stream = client.GetAppendStream(fileName))
+                using (var stream = new MemoryStream())
                 {
                     byte[] textByteArray = Encoding.UTF8.GetBytes("This is the added line.\r\n");
                     stream.Write(textByteArray, 0, textByteArray.Length);
+                    stream.Seek(0, SeekOrigin.Begin);
+                    file.Append(stream, length);
+                    file.Flush(length + stream.Length);
                 }
 
                 //Read file contents
-                using (var readStream = new StreamReader(client.GetReadStream(fileName)))
+                Response<FileDownloadInfo> fileContents = file.Read();
+                using (var readStream = new StreamReader(fileContents.Value.Content))
                 {
                     string line;
                     while ((line = readStream.ReadLine()) != null)
```

Comment:
We can replace this with `BinaryData.FromStream(fileContents.Value.Content).ToString()` and then call `Console.WriteLine` once.

Reply:
@tg-msft Updated.